django/contrib/admin/sites.py

```diff
@@ -533,16 +531,17 @@ def index(self, request, extra_context=None):
 
         return TemplateResponse(request, self.index_template or 'admin/index.html', context)
 
+    @never_cache
```

Comment:
```suggestion
    @method_decorator(never_cache)
```
This [ticket](https://code.djangoproject.com/ticket/32468#ticket) gives a thorough explanation as to why this is needed.